File: LeetCode_1748/Main.php
<?php
class Solution {

    /**
     * @param Integer[] $nums
     * @return Integer
     */
    function sumOfUnique($nums) {
        $cnt = [];
        for ($i = 1; $i <= 100; $i++) {
            $cnt[$i] =  0; 
        }

        foreach ($nums as $num) {
            $cnt[$num]++;
        }

        $sum = 0;
        for ($i = 1; $i <= 100; $i++) {
            if ($cnt[$i] === 1) {
                $sum += $i;
            }
        }

        return $sum;
    }
}
?>

File: LeetCode_1748/Main.cpp
#include<bits/stdc++.h>
using namespace std;

class Solution {
public:
    int sumOfUnique(vector<int>& nums) {
        vector<int> cnt(101, 0);
        for(int num : nums) {
            cnt[num]++;
        }

        int sum = 0;
        for(int i = 1; i <= 100; i++) {
            sum = cnt[i] == 1 ? sum + i : sum;
        }

        return sum;
    }
};
